test/test_suite_14.py

```diff
@@ -1767,6 +1767,26 @@ def test_user_agent(self):
             )
 
 
+class RouteToLeaderEnabledTest(fixtures.TestBase):
+    """
+    Check that SQLAlchemy dialect passes correct
+    route_to_leader_enabled to Client.
+    """
+
+    def setUp(self):
+        self._engine = create_engine(
+            "spanner:///projects/appdev-soda-spanner-staging/instances/"
+            "sqlalchemy-dialect-test/databases/compliance-test"
+            "?route_to_leader_enabled=False"
+        )
+
+    def test_route_to_leader_enabled(self):
```

Comment:
Can we also add an test for default option, if nothing is passed then 
`connection.connection.instance._client.route_to_leader_enabled is True`

Reply:
Added tests for enabled/disabled/default